light/errors.go

```diff
@@ -5,9 +5,42 @@ import (
 	"fmt"
 	"time"
 
+	cmtbytes "github.com/cometbft/cometbft/libs/bytes"
+	cmtmath "github.com/cometbft/cometbft/libs/math"
+	"github.com/cometbft/cometbft/light/provider"
 	"github.com/cometbft/cometbft/types"
 )
 
+var (
+
+	// ErrFailedHeaderCrossReferencing is returned when the detector was not able to cross reference the header
+	// with any of the connected witnesses.
+	ErrFailedHeaderCrossReferencing = errors.New("all witnesses have either not responded, don't have the " +
+		" blocks or sent invalid blocks. You should look to change your witnesses" +
+		"  or review the light client's logs for more information")
+	// ErrLightClientAttack is returned when the light client has detected an attempt
+	// to verify a false header and has sent the evidence to either a witness or primary.
+	ErrLightClientAttack = errors.New(`attempted attack detected.
+Light client received valid conflicting header from witness.
+Unable to verify header. Evidence has been sent to both providers.
+Check logs for full evidence and trace`,
+	)
```

Comment:
I'm not sure if it's OK to have multiline literature in error messages? You have moved earlier code, anyway, but maybe it's time to bring this to to order. No pressure, though.